visibility/test/test_sun_position/UnitTestUtilities.py

```diff
@@ -0,0 +1,29 @@
+#------------------------------------------------------------------------------
+# Copyright 2015 Esri
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#------------------------------------------------------------------------------
+
+import arcpy
+import os
+import sys
+import platform
+import logging
+
```

Comment:
Not part of this PR, but just making sure you had seen my suggestion in a [previous PR about using a named logger](https://github.com/Esri/solutions-geoprocessing-toolbox/pull/308#discussion_r38698672) - that common logger name could go in this utility file also